configurator/src/main.rs

```diff
@@ -163,6 +177,30 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let cli = Cli::parse();
 
     match cli.command {
+        Commands::Difference {
+            pictures,
```

Comment:
Please add a comment here marking that this is guaranteed to be exactly 2 elements

Reply:
Done